Feature flag `URL_REFRESH_INTERVAL` for update of url content #major (#130)
kiwigrid/k8s-sidecar: README.md

  - description: Set to true to enable pulling of 5XX response content from config map. Used in case if the filename ends with `.url` suffix (Please refer to the `*.url` feature here.)
  - required: false
  - type: boolean

> - `URL_REFRESH_INTERVAL`
>   - description: Set with integer value of seconds, where 0 means `once`. set value `> 0` to enable a dynamic update with refresh interval `URL_REFRESH_INTERVAL` of *.url content from config map. Used in case if the filename ends with `.url` suffix (Please refer to the `*.url` feature here.)
>   - required: false
>   - type: integer

Reviewer: ```suggestion
- `URL_REFRESH_INTERVAL`
  - description: How often to refresh the content of `.url` entries in seconds, where 0 means `once`.
  - required: false
  - default: 0
  - type: integer
```
